scim/acceptance/service_principal_test.go

```diff
@@ -42,3 +47,78 @@ func TestAccServicePrincipalResourceOnAws(t *testing.T) {
 		},
 	})
 }
+
+func TestAccServicePrincipalHomeDeleteSuccess(t *testing.T) {
+	if _, ok := os.LookupEnv("CLOUD_ENV"); !ok {
+		t.Skip("Acceptance tests skipped unless env 'CLOUD_ENV' is set")
+	}
+	t.Parallel()
+	appId := "12345a67-8b9c-0d1e-23fa-4567b89cde04"
+	os.Setenv("appId", appId)
+	ctx := context.Background()
+	client := common.CommonEnvironmentClient()
+	notebooksAPI := workspace.NewNotebooksAPI(ctx, client)
+	acceptance.Test(t, []acceptance.Step{
+		{
+			Template: `
+			resource "databricks_service_principal" "a" {
+				application_id = "{env.appId}"
```

Comment:
Weird I tested on AWS, GCP, and Azure and it works. Since the `application_id` is used to create the home directory, how should I reference it without passing it in?

Reply:
TODO: only set to run on azure.